src/components/Calculator.js

```diff
@@ -0,0 +1,16 @@
+import './Calculator.css';
+
+const buttonKeys = ['AC', '+/-', '%', '÷', '7', '8', '9', '×', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+
+function Calculator() {
+  return (
+    <div className="calculator-container">
+      <div className="display">0</div>
+      <div className="buttons">
+        {buttonKeys.map((key) => <button type="button" key={key}>{key}</button>)}
+      </div>
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
- [ ] The Calculator component should have at least one child component. Kindly create a child component and import it for use in this Calculator component.

- [ ] The size of buttons should also be taken into account when designing the calculator UI. Kindly improve your UI to match the design more by making the buttons smaller and not covering the entire browser width.

  ![image](https://github.com/K0ppai/Math-Magicians/assets/36542986/b8958dcc-4a00-46c0-949d-42c71706cd8d)